docs/pages/guides/notifications-services.mdx

```diff
@@ -0,0 +1,114 @@
+
+#### Considerations and limitations:
+
+- Does not support advanced platform features like notification extensions
+- Limited to 600 notifications per second per project, which works for most apps but may not suit high-volume scenarios
+
+For implementation details, see the [Expo push notifications overview](/push-notifications/overview) and [server-side SDK options](https://docs.expo.dev/push-notifications/sending-notifications/#send-push-notifications-using-a-servers).
+
+### OneSignal
+
+[OneSignal](https://onesignal.com/) is a customer engagement platform that provides push notifications, in-app messaging, SMS, and email services for web and mobile apps. It includes an [Expo SDK](https://github.com/OneSignal/onesignal-expo-plugin) for easy integration. Follow the guide below for a step-by-step setup.
+
+#### Key capabilities:
+
+- Supports [Notification Service Extensions](https://documentation.onesignal.com/docs/service-extensions) for Android and iOS
+- Provides a custom dashboard for managing notifications and user engagement
+
+#### Considerations and limitations:
+
+- Not free. OneSignal offers free limits, but advanced features (e.g., segmentation, automation) require a paid plan. See the [OneSignal pricing page](https://onesignal.com/pricing).
+- Depends on OneSignal to maintain their Expo plugin and SDK.
+
+<BoxLink
+  title="OneSignal Expo SDK Setup"
+  description="Learn how to integrate OneSignal in your Expo project."
+  href="https://documentation.onesignal.com/docs/react-native-expo-sdk-setup"
+/>
+
+### Braze
+
+[Braze](https://www.braze.com/) is a customer engagement platform that delivers personalized, cross-channel messaging through push notifications, in-app messaging, email, SMS, and web. It provides a [React Native SDK](https://github.com/braze-inc/braze-react-native-sdk) and an [Expo plugin](https://github.com/braze-inc/braze-expo-plugin/tree/main). Check out the [Expo example app](https://github.com/braze-inc/braze-expo-plugin/tree/main/example) for more details.
+
+#### Key capabilities:
+
+- Supports customizable push notifications with images, videos, and location
+- Includes a dashboard for managing campaigns and journey orchestration
+
+#### Considerations and limitations:
+
+- Requires contacting Sales for pricing information
+- Depends on Braze to maintain their Expo plugin and SDK
+
+<BoxLink
+  title="Braze Expo Setup"
+  description="Learn how to integrate Braze in your Expo project."
+  href="https://www.braze.com/docs/developer_guide/platforms/react_native/sdk_integration"
+/>
+
+### Customer.io
+
+[Customer.io](http://Customer.io) provides two main services: **Journeys**, for setting up campaigns and sending messages, emails, and push notifications, and **Data Pipelines**, for capturing and forwarding data from your Expo app, website, or other sources to the right destinations in your stack. It offers an [Expo plugin](https://github.com/customerio/customerio-expo-plugin) for seamless integration.
```

Comment:
```suggestion
[Customer.io](http://Customer.io) provides two main services: 
- **Journeys**, for setting up campaigns and sending messages, emails, and push notifications
- **Data Pipelines**, for capturing and forwarding data from your Expo app, website, or other sources to the right destinations in your stack

It offers an [Expo plugin](https://github.com/customerio/customerio-expo-plugin) for seamless integration.
```

both features have multiple capabilities so let's cover it into list like this for easier readability